<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgHotspotCreate</class>
 <widget class="QDialog" name="DlgHotspotCreate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>432</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="lbLeftup">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>120</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbLeftupIcon">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbLeftupTitle">
   <property name="geometry">
    <rect>
     <x>34</x>
     <y>6</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>215</x>
     <y>210</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOk">
   <property name="geometry">
    <rect>
     <x>315</x>
     <y>210</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbSecurity">
   <property name="geometry">
    <rect>
     <x>76</x>
     <y>60</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="cbxSecurity">
   <property name="geometry">
    <rect>
     <x>175</x>
     <y>55</y>
     <width>200</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbNetName">
   <property name="geometry">
    <rect>
     <x>76</x>
     <y>105</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="leNetName">
   <property name="geometry">
    <rect>
     <x>175</x>
     <y>100</y>
     <width>200</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbPassword">
   <property name="geometry">
    <rect>
     <x>76</x>
     <y>150</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lePassword">
   <property name="geometry">
    <rect>
     <x>175</x>
     <y>145</y>
     <width>200</width>
     <height>32</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxPwd">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>157</y>
     <width>18</width>
     <height>9</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
